cookbook/case_studies/ml_training/spark_horovod/keras_spark_rossmann_estimator.py

```diff
@@ -303,13 +343,15 @@ def build_vocabulary(df: pyspark.sql.DataFrame) -> Dict[str, List[Any]]:
         vocab[col] = sorted(values, key=lambda x: x or default_value)
     return vocab
 
-
+# %%
+# We define a function that casts continuous columns to float and returns it as a DataFrame.
 def cast_columns(df: pyspark.sql.DataFrame, cols: List[str]) -> pyspark.sql.DataFrame:
     for col in cols:
         df = df.withColumn(col, F.coalesce(df[col].cast(T.FloatType()), F.lit(0.0)))
     return df
 
-
+# %%
+# Next, we define a function that looks up for categorical columns and returns it as a DataFrame.
```

Comment:
Can we rephrase this line? I don't think it's clear. Please explain what "look up" does vaguely.